src/tool/subcommands/api_cmd.rs

```diff
@@ -113,41 +132,28 @@ pub enum ApiCommands {
         /// Cancel test run on the first failure
         #[arg(long)]
         fail_fast: bool,
-        #[arg(short, long, default_value = "10")]
-        /// The number of tipsets to use to generate test cases.
-        n_tipsets: usize,
+
         #[arg(long, value_enum, default_value_t = RunIgnored::Default)]
         /// Behavior for tests marked as `ignored`.
         run_ignored: RunIgnored,
         /// Maximum number of concurrent requests
         #[arg(long, default_value = "8")]
         max_concurrent_requests: usize,
-        /// Miner address to use for miner tests. Miner worker key must be in the key-store.
+
+        #[command(flatten)]
+        create_tests_args: CreateTestsArgs,
```

Comment:
nit: there's no easy way to figure out what `CreateTestsArgs` does without navigating to the actual type, let's add a comment explaining it's purpose.

Reply:
Can you suggest what the docs would be? These are the args for the `creat_tests` function, so I think both the variable name and the type name are pretty descriptive. The CLI user will still get the correct `--help`

Reply:
Just as long as users can figure out where the snapshot goes. 

It was quite confusing for me that the snapshot path is hidden behind this as someone already familiar with the code.